DuckDuckGo/AppServices/AutofillService.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  AutofillService.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import BrowserServicesKit
+import Core
+import Common
+
+final class AutofillService {
+
+    private let autofillLoginSession = AppDependencyProvider.shared.autofillLoginSession
+    private let autofillUsageMonitor = AutofillUsageMonitor()
+    private var autofillPixelReporter: AutofillPixelReporter?
+
+    var syncService: SyncService?
+
+    init() { // TODO: this was always done after DB was initialized, so looks like this code was never being executed, to confirm
```

Comment:
Since it's a part of a protocol, I'd assume the bool property (`isDatabaseFileInitialized`) can be false in some corner cases, so I'd say it's reasonable to have such check.

Reply:
turns out it can be removed too:
https://app.asana.com/0/0/1209194443865280/f